### PR TITLE
Initial port

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr review please

File: ratings/serializers.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                  'updated_at')
+
+
+class RatingSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Rating
+        read_only_fields = ('created_by', 'updated_by')
+        fields = ('id', 'identity', 'invite', 'version', 'question_id',
+                  'question_text', 'answer_text', 'answer_value', 'created_at')
```
Comment (gsvr) [inline ratings/serializers.py:22]: Did you leave off updated_at on purpose?

File: ratings/serializers.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from .models import Invite, Rating
+from rest_hooks.models import Hook
+from rest_framework import serializers
+
+
+class InviteSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Invite
+        read_only_fields = ('created_by', 'updated_by')
```
Comment (gsvr) [inline ratings/serializers.py:10]: Do you want to include `id, created_at, updated_at` as read_only?

File: ratings/views.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+class RatingViewSet(viewsets.ModelViewSet):
+
+    """
+    API endpoint that allows dummy models to be viewed or edited.
+    """
+    permission_classes = (IsAuthenticated,)
+    queryset = Rating.objects.all()
+    serializer_class = RatingSerializer
+    filter_fields = ('identity', 'invite', 'version', 'question_id',
+                     'created_at')
```
Comment (gsvr) [inline ratings/views.py:49]: `update_at` was left of here as well, so I'm starting to think it was on purpose.

**Comment by gsvr** [comment]
Just a few questions, :+1: when addressed.

File: ratings/serializers.py (hunk 1 above)
Comment (imsickofmaps) [inline ratings/serializers.py:22]: Yeah, these shouldn't be updated

File: ratings/serializers.py (hunk 2 above)
Comment (imsickofmaps) [inline ratings/serializers.py:10]: They're auto fields

**Pull request merged**